ibazel/ibazel.go

```diff
@@ -127,33 +128,37 @@ func (i *IBazel) handleSignals() {
 	switch sig {
 	case syscall.SIGINT:
 		if i.cmd != nil && i.cmd.IsSubprocessRunning() {
-			fmt.Fprintf(os.Stderr, "\nSubprocess killed from getting SIGINT (trigger SIGINT again to stop ibazel)\n")
+			fmt.Fprint(os.Stderr, "\n")
+			log.Log("Subprocess killed from getting SIGINT (trigger SIGINT again to stop ibazel)\n")
 			i.cmd.Terminate()
 		} else {
 			osExit(3)
 		}
 		break
 	case syscall.SIGTERM:
 		if i.cmd != nil && i.cmd.IsSubprocessRunning() {
-			fmt.Fprintf(os.Stderr, "\nSubprocess killed from getting SIGTERM\n")
+			fmt.Fprint(os.Stderr, "\n")
+			log.Log("Subprocess killed from getting SIGTERM\n")
 			i.cmd.Terminate()
 		}
 		osExit(3)
 		return
 	case syscall.SIGHUP:
 		if i.cmd != nil && i.cmd.IsSubprocessRunning() {
-			fmt.Fprintf(os.Stderr, "\nSubprocess killed from getting SIGHUP\n")
+			fmt.Fprint(os.Stderr, "\n")
```

Comment:
It feels a bit weird to have to fallback to the old mechanism of logging whenever you want to prefix your message with a newline. Maybe you could just add a helper method to the logger?

`log.NewLine()` perhaps?

Reply:
Alternatively you could have each logger method take an optional integer argument of how many newlines to prefix a message with.

Reply:
I think you're on to something here. Log should append a \n at the end of the line on its own which will un-clutter most of this.